CITATION.cff

```diff
@@ -0,0 +1,36 @@
+# YAML 1.2
+---
+abstract: "Pydantic is the most widely used data validation library for Python."
+authors:
+  - family-names: Colvin
```

Comment:
I think we can just do a list (don't have to separate family and given name)

Reply:
all authors mentioned here were mentioned. https://docs.pydantic.dev/latest/pydantic_people/

Reply:
kindly check the latest file .